docs/admin/upgrading.rst

```diff
@@ -37,6 +37,15 @@ Basic instructions
    Any modifications should have been written within plugins, so that they are not lost on overwriting.
    If this is not the case, take care to maintain your modifications. 
 
+From 2.x to 3.0
```

Comment:
changed

I copied the header from /docs/guids/upgrading.rst, that why that was